config/globals.js

```diff
@@ -24,12 +24,15 @@ switch (NODE_ENV) {
 }
 
 let defaultDbUrl = null;
+let defaultTldrawDbUrl = null;
 switch (NODE_ENV) {
 	case ENVIRONMENTS.TEST:
 		defaultDbUrl = 'mongodb://127.0.0.1:27017/schulcloud-test';
+		defaultTldrawDbUrl = 'mongodb://127.0.0.1:27017/tldraw-test';
```

Comment:
We use for nestJS a in memory mongoDB. Please remove this line and check nestjs nest and test-modules.